frontend/src/pages/account-settings/components/subscription/subscription.tsx

```diff
@@ -42,6 +45,7 @@ const Subscription: React.FC = () => {
           onCheckout={handleSubscribe}
           price={SUBSCRIPTION_PRICE}
           label="Subscribe"
+          disabled={Boolean(user?.isSubscribed)}
```

Comment:
This may be changed after PR with displaying active subscription date

Reply:
on this page user should always be present. so you need to avoid an optional changing